Source: Based on debian patch
Upstream: no
Reason: Use lxterminal in quicklauch bar not gnome-terminal.
--- lxpanel-0.5.3/data/default/panels/panel.in~	2009-09-20 00:00:19.963895061 +0800
+++ lxpanel-0.5.3/data/default/panels/panel.in	2009-09-20 00:01:13.264503988 +0800
@@ -53,7 +53,7 @@
             id=pcmanfm.desktop
         }
         Button {
-            id=gnome-terminal.desktop
+            id=lxterminal.desktop
         }
         Button {
             id=firefox.desktop
